Runtime/Scripts/RGBotLocalRuntime/SampleBot/RGSampleBot.cs

```diff
@@ -38,17 +38,21 @@ public override void ProcessTick(RG rgObject)
                 {
                     var target = entities[new Random().Next(entities.Count)];
 
-                    var targetPosition = target.position ?? Vector3.zero;
-                    //TODO (REG-1302): If Actions were strongly typed we wouldn't need to build this weird map...
-                    var action = new RGActionRequest("PerformSkill", new Dictionary<string, object>
+                    // ensure target is not UI component
+                    if (target.ContainsKey("position"))
```

Comment:
I'm a bit confused by this, won't position always be defined since UI components have a rect transform?

Reply:
i agree with you abby, I would expect this check to be un-necessary